src/server/cluster/cluster_family.cc

```diff
@@ -25,12 +25,15 @@
 #include "server/server_family.h"
 #include "server/server_state.h"
 
-ABSL_FLAG(std::string, cluster_announce_ip, "", "ip that cluster commands announce to the client");
+ABSL_FLAG(std::string, cluster_announce_ip, "", "DEPRECATED: use --announce_ip");
```

Comment:
`ABSL_RETIRE_FLAG` ?

Reply:
Retiring a flag means we can't access it, which is not desired. We'd like to maintain backward compatibility for 2+ releases.
Note that we currently also use this flag in our managed environment